docs/UserGuide.adoc

```diff
@@ -310,9 +316,17 @@ Examples:
 
 ==== Viewing an eatery: `show`
 
-Shows an eatery with all its details i.e. address, category, tags, reviews, external reviews (from Google), and a map of its location.
+Shows an eatery with all its details - address, tags, reviews, and a map of its location.
+
 Format: `show [id]`
 
+****
+* Shows the eatery at the specified index.
+* The index refers to the index number shown in the displayed eatery list.
+* The index *must be a positive integer* (e.g. 1, 2, 3, ...).
```

Comment:
Can delete the "The index must be a positive integer (e.g. 1, 2, 3, ...)." part bc I added it in the "Command format" section of the UG in my latest PR bc I realised a lot of the commands keep repeating this part lel

Reply:
ok!